### PR TITLE
Fix/disconnection #57

### PR DESCRIPTION
**Pull request by Toine-ddt**

### CODE REVIEW

File: src/IpAdress/ScoreIpAdress.qml [hunk 1]
```diff
@@ -1,3 +1,11 @@
+/*
+  * At the top left of the interface with others buttons
+  * Contain ScoreConnection and ScoreDisconnection :
+  * - ScoreConnection : a window which is asking for ip adress
+  * - ScoreConnection : a window for disconnection
```
Comment (jcelerier) [inline src/IpAdress/ScoreIpAdress.qml:5]: ScoreDisconnection

File: src/PlayPauseStop/ScorePlayPause.qml [hunk 2]
```diff
@@ -1,3 +1,16 @@
+/*
+  * Play and Pause buttons :
+  * - at the top left of the interface with others buttons
+  * - visibles when the remote is connected to score
```
Comment (jcelerier) [inline src/PlayPauseStop/ScorePlayPause.qml:4]: use a spell checker :) visible, synchronise, ...

Comment (jcelerier) [reply src/PlayPauseStop/ScorePlayPause.qml:4]: https://github.com/CJCombrink/SpellChecker-Plugin

File: src/Skeleton/ScoreSkeleton.qml [hunk 3]
```diff
@@ -61,7 +73,7 @@ Item {
     /*
     // TODO : Creating the volume slider object
     ScoreVolume {
-        id: scoreVolume
+        id: scoreVolumeS
```
Comment (jcelerier) [inline src/Skeleton/ScoreSkeleton.qml:76]: why the S in capital ? isn't that a typo or cat that walked on the keyboard ?

Comment (jcelerier) [reply src/Skeleton/ScoreSkeleton.qml:76]: it's especially suspicious considering that nothing else in this commit seems to refer to this id

**Review by jcelerier — CHANGES_REQUESTED** [review]

File: src/Skeleton/ScoreSkeleton.qml (hunk 3 above)
Comment (Toine-ddt) [reply src/Skeleton/ScoreSkeleton.qml:76]: I think it is because of a wrong ctrl-s ctrl-r :)

**Comment by Toine-ddt** [comment]
conflicts resolved

**Pull request merged**